### ISSUE TITLE
Track new rst files in git

### ISSUE THREAD

**Issue by damonbayer** [CDCgov/multisignal-epi-inference#194]
Intentially. At some point, we only wanted to keep track of `index.rst`, but when new tutorials are added, `git status` was not reporting new `rst` files generated by `.pre-commit-rst-placeholder.sh`.

Now that I think of it, perhaps we could modify the script to also add newly generated `rst` files via `git add`?

_Originally posted by @gvegayon in https://github.com/CDCgov/multisignal-epi-inference/pull/158#discussion_r1637056061_

**Comment by gvegayon** [comment]
@damonbayer, flagging this for your awareness. Is this still relevant?

**Comment by damonbayer** [comment]
Yes, but the tutorials will get overhauled when we implement https://github.com/CDCgov/PyRenew/issues/231.
